README.md

```diff
@@ -57,8 +46,8 @@ Vita3K is licensed under the **GPLv2** license. This is largely dictated by exte
 Please see [`building.md`](./building.md).
 
 ## Running
-Specify the path to a .vpk file as the first command line argument, or run `Vita3K --help` from the command-line for a full list of options.
-For more detailed instructions on running/installing games on all platforms, please read the **#info-faq** channel on our [Discord Server](https://discord.gg/MaWhJVH).
+Check our [quickstart](https://vita3k.org/quickstart) guide to make sure your computer meets the minimum requirements to run Vita3K.  
```

Comment:
This is **really** petty but the link should cover the full "quickstart guide" instead of just "quickstart".